### ISSUE TITLE
UIAElement.isStale:scrollToVisible() is causing problems

### ISSUE THREAD

**Issue by rsalvador** [ios-driver/ios-driver#16]
scrollToVisible() is called when doing read-like actions on an element i.e. element.getAttribute() or element,findElement() and this causes problems if the element is not visible (behind some other element). Also doing an element.getAttribute("..") makes the element scroll into view which is not what one would expect.

**Comment by akusnadi** [comment]
Yes, I am also running into the same issue. Basically, I am trying to call getText() to get the text within an element after calling getAttribute() and this causes the test to fail to get the text because it is not able to scroll into the element.
